books/free-programming-books-langs.md

```diff
@@ -1316,6 +1316,7 @@ Books on general-purpose programming that don't focus on a specific language are
 * [Essential Node.js](https://www.programming-books.io/essential/nodejs/) - Krzysztof Kowalczyk, StackOverflow Contributors
 * [From Containers to Kubernetes with Node.js](https://www.digitalocean.com/community/books/from-containers-to-kubernetes-with-node-js-ebook) - Kathleen Juell (PDF, EPUB)
 * [Full Stack JavaScript: Learn Backbone.js, Node.js and MongoDB](https://github.com/azat-co/fullstack-javascript) - Azat Mardan
+* [Full Stack Node.js: The Complete Guide to Building Production Apps with Node.js](https://moscow-city.guide/upload/iblock/8ce/8ceee361e68ff14b4e2aa66575be659d.pdf) - David Gutman, Nate Murray (PDF)
```

Comment:
This is not an authorized distribution. Refer to docs/CONTRIBUTING.md for guidelines.

Reply:
I think you are talking about the point 4 which is given below:
* use the link with the most authoritative source (meaning the author's website is better than the editor's website, which is better than a third-party website)

This point showing us a priority, not denying to not upload the third-party website. Right?